tests/workspace_svg/workspace_svg_test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Blockly Tests
+ *
+ * Copyright 2017 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+function helper_createWorkspaceWithToolbox() {
+  var toolbox = document.getElementById('toolbox-categories');
+  return Blockly.inject('blocklyDiv', {toolbox: toolbox});
```

Comment:
I'm okay with this for now because yay. tests! I'm curious, though, what kind of problems we run into with initialization if we create the WorkspaceSvg object via the constructor instead of inject?